src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -162,6 +163,29 @@ public Manifests(final Map<String, String> attrs) {
         this.attributes = new ConcurrentHashMap<String, String>(attrs);
     }
 
+    /**
+     * Reads all values with specified key from several manifests.
```

Comment:
@nesterov-n I think it should be more descriptive. What if there is no such key in any manifest or there is same key with same/different values across manifests?